### DISCUSSION TITLE
How can I obtain the 2D skeleton from Heatmap Regression Model

### DISCUSSION THREAD

**Discussion by Pan-Mengyu** [Nicholasli1995/EvoSkeleton#55] [Q&A] [answered]
Hello, I'm using the pre-trained model in my own images and it seems to update example_annot.py. 

What I do is that I run  python ../tools/imgto2Dnet.py --cfg "./h36m2Dpose/cfgs.yaml" but it cannot find h36m_annot_train.npy. 

The error : 
FileNotFoundError: [Errno 2] No such file or directory: '/h36m/h36m_annot_train.npy'

What should I do to obtain the 2D skeleton from Heatmap Regression Model?

**Comment by Nicholasli1995** [comment, marked as answer]
To use the pre-trained model, please follow the instructions on https://github.com/Nicholasli1995/EvoSkeleton/blob/master/docs/2DHPE.md. You should use python h36m2Dpose.py. You can preprare your own image similar to the examples.

**Reply by Pan-Mengyu** [reply]
Thank you, I will try it
